docs/3.api/1.components/13.youtube-embed.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "<YoutubeEmbed>"
+description: A simple and performant YouTube component.
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/scripts-and-assets/blob/main/modules/nuxt-third-party-capital/src/runtime/components/YoutubeEmbed.ts
+    size: xs
+---
+
+The YoutubeEmbed component is based on [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed).
```

Comment:
Nit: Can we expand this a bit to explain what it does first? E.g:

"The YouTubeEmbed component can be used to load display a YouTube embed. It uses lite-youtube-embed to load significantly faster"